### DISCUSSION TITLE
Button Table/Grid View

### DISCUSSION THREAD

**Discussion by antoinemy** [filamentphp/filament#5410] [Help] [answered]
Hello,

Is it possible to add a custom button or toggle to switch from a table view to a grid view? I discovered this feature in the documentation: https://filamentphp.com/docs/2.x/tables/layout#arranging-records-into-a-grid

Imagine a button at the top right of the table to change views without impacting filters/sorting/searchs?

I couldn't find a topic in the threads on your GitHub?

Thanking you

**Comment by danharrin** [comment, marked as answer]
I think that's way too complex to implement and design an API for unfortunately
